Source/WebKit/UIProcess/Cocoa/ProcessAssertionCocoa.mm

```diff
@@ -389,6 +393,22 @@ - (void)assertion:(RBSAssertion *)assertion didInvalidateWithError:(NSError *)er
     };
 }
 
+ProcessAssertion::ProcessAssertion(AuxiliaryProcessProxy& process, const String& reason, ProcessAssertionType assertionType)
+    : m_assertionType(assertionType)
+    , m_pid(process.processID())
+    , m_reason(reason)
+{
+#if USE(EXTENSIONKIT_ASSERTIONS)
+    NSString *runningBoardAssertionName = runningBoardNameForAssertionType(assertionType);
+    NSString *runningBoardDomain = runningBoardDomainForAssertionType(assertionType);
+    auto capability = adoptNS([_SECapabilities assertionWithDomain:runningBoardDomain name:runningBoardAssertionName]);
+    _SEExtensionProcess *extensionProcess = process.extensionProcess();
```

Comment:
Doesn't `process.extensionProcess()` return a `RetainPtr<_SEExtensionProcess>`?

Seems like this should be `auto extensionProcess = process.extensionProcess()`